queries.sql

```diff
@@ -6,3 +6,29 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Vet clinic database: query and update animals table
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE name LIKE IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;

```

Comment:
- [ ] Great job on your queries! 😊 However, I noticed a couple of improvements that can enhance the quality of your code:

     1. **Add the `BEGIN` Transaction Keyword**: Your query appears to be missing the `BEGIN` keyword at the start of the transaction. To ensure completeness and correctness, please add the `BEGIN` keyword at the top of your transaction.

    2. **Enhance Readability with Spacing**: To improve code readability and maintainability, consider adding at least one empty space after each complete query. This simple practice can make your code cleaner and easier to follow.

By incorporating these suggestions, your code will not only be complete but also more aesthetically pleasing and easier for both you and code reviewers to work with. 

Happy coding! 😊🚀🍀